drivers/tests/TESTS/mbed_drivers/lp_ticker/main.cpp

```diff
@@ -117,19 +128,18 @@ void test_multi_ticker(void)
 void test_multi_call_time(void)
 {
     LowPowerTicker ticker;
-    int time_diff;
     const int attach_count = 10;
 
     for (int i = 0; i < attach_count; i++) {
         ticker_callback_flag = 0;
         gtimer.reset();
 
         gtimer.start();
-        ticker.attach_us(callback(stop_gtimer_set_flag), MULTI_TICKER_TIME_MS * 1000);
+        ticker.attach(callback(stop_gtimer_set_flag), MULTI_TICKER_TIME);
         while (!ticker_callback_flag);
-        time_diff = gtimer.read_us();
+        auto time_diff = gtimer.elapsed_time();
 
-        TEST_ASSERT_UINT32_WITHIN(TOLERANCE_US(MULTI_TICKER_TIME_MS * 1000), MULTI_TICKER_TIME_MS * 1000, time_diff);
+        TEST_ASSERT_DURATION_WITHIN(TOLERANCE_US(MULTI_TICKER_TIME), MULTI_TICKER_TIME, time_diff);
```

Comment:
Is this correct? The tolerance still requires microseconds and MULTI_TICKER_TIME is in milliseconds ? 
ie #define TOLERANCE_US(DELAY) (600us + DELAY / 20) 

So you are effectively doing 600us + 100ms/20 
Not sure what this will do when comparing against time_diff ? The chrono library may just handle it automatically but I'm not sure...

Reply:
`600us + 100ms/20` is indeed fine, chrono will handle necessary common unit conversion. However the name `TOLERANCE_US` is definitely misleading, I will rename to a unit agnostic `TOLERANCE`.